front_end/webserver.py

```diff
@@ -392,6 +394,9 @@ def post(self):
                         result = f"Success: The user '{delete_user}' has been deleted."
                 else:
                     result = f"Error: The user '{delete_user}' does not exist."
+            else:
+                if not remove_user:
+                    result = f"Error: Please enter a user."
```

Comment:
fixes bug where CodeBuddy crashes when submit is pressed without entering a user